README.rst

```diff
@@ -90,6 +90,19 @@ At the command line::
     pip install magpie
 
 
+Installation
+============
```

Comment:
for future ref, specify in https://github.com/Ouranosinc/Magpie/blob/master/docs/installation.rst